naxxatrapy/naxxatrapy.py

```diff
@@ -1,3 +1,6 @@
+G = 6.674 × 10**(-11)
+from numpy import *
```

Comment:
Import statement should always be at the top of the file

Reply:
Ok will do the changes.